src/ui/components/Rating/index.js

```diff
@@ -68,6 +68,7 @@ export class RatingBase extends React.Component {
         id: `Rating-rating-${thisRating}`,
         key: `rating-${thisRating}`,
         ref: (ref) => { this.ratingElements[thisRating] = ref; },
+        title: `Rate this add-on ${thisRating} out of 5.`,
```

Comment:
This should be <code>title: readOnly ? \`Rate this add-on ${thisRating} out of 5.\` : null,</code>